### ISSUE TITLE
Horizontal scrolling is not reversed

### ISSUE THREAD

**Issue by rituraj22** [Genymobile/scrcpy#49]
Vertical scrolling is reversed, but horizontal is not. So scrolling is confusing.

**Comment by rom1v** [comment]
Thank you for the report. I currently have no way to test horizontal scrolling.

However, the [documentation](https://wiki.libsdl.org/SDL_MouseWheelEvent#Remarks) says:

> Movements to the left generate negative x values and to the right generate positive x values. Movements down (scroll backward) generate negative y values and up (scroll forward) generate positive y values.

This seems inconsistent to me.

Since the vertical scrolling works as expected (same behavior as on desktop environments on Linux), we should reverse the horizontal scrolling.

Could you apply this change and confirms it is ok to you, please?

```diff
--- a/app/src/convert.c
+++ b/app/src/convert.c
@@ -172,7 +172,7 @@ SDL_bool mouse_wheel_from_sdl_to_android(const SDL_MouseWheelEvent *from,
     to->scroll_event.position = position;
 
     int mul = from->direction == SDL_MOUSEWHEEL_NORMAL ? 1 : -1;
-    to->scroll_event.hscroll = mul * from->x;
+    to->scroll_event.hscroll = -mul * from->x;
     to->scroll_event.vscroll = mul * from->y;
 
     return SDL_TRUE;
```

**Comment by rituraj22** [comment]
@rom1v Multiplying by -1 did work correctly (horizontal scrolling became reversed).
FYI, I am testing it in webview on
Micromax Canvas A1 (AQ4501)
sprout4
Android 6.0.1

**Comment by rituraj22** [comment]
Log shows (-1,0) on both devices when i scroll to right. But the motion on both devices are opposite.
On the phone with 6.0, (-1,0) scrollls to right, while on 7.1, (-1,0) scrolls to left.

**Comment by rom1v** [comment]
@rituraj22 Could you provide the logs as printed please? (I also need the mul value) ;)

**Comment by rituraj22** [comment]
INFO: WHEEL mul=1; (-1, 0)
is on all cases on scroll to right

**Comment by rom1v** [comment]
> On the phone with 6.0, (-1,0) scrollls to right

This is very unexpected.

Just for confirmation, it does not print:

    INFO: WHEEL mul=-1; (-1, 0)

but really prints:

    INFO: WHEEL mul=1; (-1, 0)

?

**Comment by rituraj22** [comment]
Yeah on all devices it prints 
``
[0/1] Running external command run.
/home/cogito/testings/scrcpy(copy)/x/server/scrcpy-server.jar: 1 file pushed. 4.6 MB/s (19038 bytes in 0.004s)
INFO: Initial texture: 480x854
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (0, 1)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
INFO: WHEEL mul=1; (-1, 0)
``

**Comment by rom1v** [comment]
I said:

> On the phone with 6.0, (-1,0) scrollls to right
> 
>> This is very unexpected.

In fact, since I reversed the horizontal direction, the unexpected behavior is:

> while on 7.1, (-1,0) scrolls to left.

**Comment by rom1v** [comment]
@rituraj22 Please confirm that https://github.com/Genymobile/scrcpy/issues/966#issuecomment-559894169 does not break the behavior you want.

**Comment by rom1v** [comment]
Was fixed by #966.
